opentelemetry-sdk/src/opentelemetry/sdk/_metrics/sdk_configuration.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from typing import Sequence
+
+from opentelemetry.sdk._metrics.metric_reader import MetricReader
+from opentelemetry.sdk.resources import Resource
+
+
+@dataclass
+class SdkConfiguration:
+    resource: Resource
+    # TODO: once views are added
+    # views: Sequence[View]
+    metric_readers: Sequence[MetricReader]
```

Comment:
Do we really need this class? It looks like just a wrapper for several different objects and its name is hardly descriptive of them all.

Reply:
What's the alternative?

Reply:
Just pass the resource and metric readers as normal arguments to the measurement consumer.

Reply:
So I did that originally in my prototype but ended up needing to pass info to deeper levels e.g. views need to be known when creating aggregators. It is easier to wrap them up like this to pass them down and you don't have to change the parameters at every level. It has the added benefit that any changes made to the config object (like updating resource) will be visible to anyone with a reference.